### DISCUSSION TITLE
how to sign in on google playstore please?

### DISCUSSION THREAD

**Discussion by ahpoh1349** [undergroundwires/privacy.sexy#404] [Q&A]
hi respected devs,how to sign in to my google play store?..cant sign in for now and the restore point i made earlier also gone,also now i cant update my windows update too but i didnt click on that..please kindly help thanks..any walk to do it please?after i did the privacy.sexy app thing on my pc, thanks

**Comment by undergroundwires** [comment]
Hi, you can [create a "Script Issues" issue](https://github.com/undergroundwires/privacy.sexy/issues/new/choose) instead, it will guide you to provide more information so you can get better help.

**Reply by ahpoh1349** [reply]
thanks will do!
